src/System.Windows.Forms/src/System/Windows/Forms/DataGridViewSelectedRowCollection.cs

```diff
@@ -28,16 +26,16 @@ void IList.Clear()
             throw new NotSupportedException(SR.DataGridView_ReadOnlyCollection);
         }
 
-        bool IList.Contains(object value) => ((IList)_items).Contains(value);
+        bool IList.Contains(object? value) => ((IList)_items).Contains(value);
 
-        int IList.IndexOf(object value) => ((IList)_items).IndexOf(value);
+        int IList.IndexOf(object? value) => ((IList)_items).IndexOf(value);
 
-        void IList.Insert(int index, object value)
+        void IList.Insert(int index, object? value)
         {
             throw new NotSupportedException(SR.DataGridView_ReadOnlyCollection);
         }
```

Comment:
We can clean these up a little more by doing the following in the appropriate places
```suggestion
        void IList.Insert(int index, object? value) => throw new NotSupportedException(SR.DataGridView_ReadOnlyCollection);
```